src/main/java/com/cinema/model/dto/mapper/TicketDtoMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.cinema.model.dto.mapper;
+
+import com.cinema.model.Ticket;
+import com.cinema.model.dto.TicketResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TicketDtoMapper {
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public TicketDtoMapper(MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    public TicketResponseDto mapTicketToDto(Ticket ticket) {
```

Comment:
```suggestion
    public TicketResponseDto mapToDto(Ticket ticket) {
```
uniform method names